Health/src/main/webapp/app/controller/dataframe/CancerData.js

```diff
@@ -20,31 +20,108 @@ function getColumnName(){
 	var DataFrame = dfjs.DataFrame;
```

Comment:
Guys Please make classes and try to move this code in class structure.
Let me know if you need any help